packages/dds/tree/src/test/simple-tree/api/treeNodeApi.spec.ts

```diff
@@ -1239,6 +1240,45 @@ describe("treeNodeApi", () => {
 			const a = TreeAlpha.create(A, { x: 1 });
 			assert.deepEqual(a, { x: 1 });
 		});
+
+		it("object with defaulted identifier field", () => {
+			const A = schema.object("A", { x: schema.identifier });
+			const node = TreeAlpha.create(A, { x: undefined });
+
+			// TODO: make this work instead of error:
+			// assert(isStableId(node.x));
```

Comment:
I'm still working on these cases, but I figured that work should be separated, but might as well land the tests for it now.